core/src/main/java/de/danoeh/antennapod/core/storage/DBUpgrader.java

```diff
@@ -310,6 +310,10 @@ static void upgrade(final SQLiteDatabase db, final int oldVersion, final int new
             db.execSQL("ALTER TABLE " + PodDBAdapter.TABLE_NAME_FEEDS +
                     " ADD COLUMN " + PodDBAdapter.KEY_FEED_SKIP_ENDING + " INTEGER DEFAULT 0;");
         }
+        if (oldVersion < 1090001) { // fixme / todo: fix  version
```

Comment:
It's stored in `PodDBAdapter.VERSION`. I usually use the expected release version code for that change. As this will be released in AntennaPod 2.2.0, the code would be `2020000`.